<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1337</width>
    <height>506</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>汉仪文黑-85W</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>原神15s速切模型</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>1281</width>
      <height>401</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>伤害计算</string>
     </attribute>
     <widget class="QTableWidget" name="tbl_2">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>60</y>
        <width>621</width>
        <height>211</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::NoPen</enum>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>18</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>137</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>23</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>41</number>
      </attribute>
      <row>
       <property name="text">
        <string>伤害</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>理之冠</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>时之砂</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>空之杯</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>第一</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>第二</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>第三</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>第四</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>280</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_name">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_run">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>260</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>计算</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_refine">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>180</y>
        <width>87</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="cb_cnum">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>60</y>
        <width>87</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_skill">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>110</y>
        <width>87</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>110</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>技能等级</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_wp">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>60</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>命座</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>180</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>精炼</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_load_wp">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>10</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>读取Log</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="rb_pop">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>141</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>速切</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="rb_display">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>30</y>
        <width>141</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>伤害精确显示</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_action">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>260</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>技能参数</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>290</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label2">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>280</y>
        <width>311</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_buff">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>300</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>增益参数</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>圣遗物主词条</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>201</width>
        <height>291</height>
       </rect>
      </property>
      <property name="title">
       <string>理之冠</string>
      </property>
      <widget class="QCheckBox" name="cbox_h_cr">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>91</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>暴击</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbox_h_cd">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>91</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>暴伤</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbox_h_ar">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>91</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>攻击</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbox_h_dr">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>91</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>防御</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbox_h_em">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>91</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>精通</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>201</width>
        <height>291</height>
       </rect>
      </property>
      <property name="title">
       <string>时之沙</string>
      </property>
      <widget class="QCheckBox" name="cbox_g_ar">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>91</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>攻击</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbox_g_dr">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>91</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>防御</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbox_g_em">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>91</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>精通</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>20</y>
        <width>201</width>
        <height>291</height>
       </rect>
      </property>
      <property name="title">
       <string>空之杯</string>
      </property>
      <widget class="QCheckBox" name="cbox_c_ed">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>91</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>属伤</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbox_c_dphys">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>91</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>物伤</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbox_c_ar">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>91</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>攻击</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbox_c_em">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>91</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>精通</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="b_read_main">
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>250</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>读取</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b_save_main">
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>290</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>存储</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>其他增益</string>
     </attribute>
     <widget class="QPushButton" name="b_save">
      <property name="geometry">
       <rect>
        <x>1130</x>
        <y>300</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>存储</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b_read">
      <property name="geometry">
       <rect>
        <x>1130</x>
        <y>260</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>读取</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>291</width>
        <height>351</height>
       </rect>
      </property>
      <property name="title">
       <string>圣遗物副词条</string>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>暴击</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="digit_cr">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>40</y>
         <width>70</width>
         <height>22</height>
        </rect>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>240</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>防御（固定）</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="digit_cd">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>80</y>
         <width>70</width>
         <height>22</height>
        </rect>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="digit_ar">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>120</y>
         <width>70</width>
         <height>22</height>
        </rect>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="digit_dr">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>200</y>
         <width>70</width>
         <height>22</height>
        </rect>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>200</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>防御(%)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>攻击（固定）</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>攻击(%)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>暴伤</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="digit_sa">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>160</y>
         <width>70</width>
         <height>22</height>
        </rect>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>280</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>生命(%)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>320</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>生命（固定）</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>10</y>
        <width>291</width>
        <height>351</height>
       </rect>
      </property>
      <property name="title">
       <string>其他加成</string>
      </property>
      <widget class="QDoubleSpinBox" name="digit_fd">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>80</y>
         <width>70</width>
         <height>22</height>
        </rect>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>属伤</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>物伤</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="digit_ed">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>40</y>
         <width>70</width>
         <height>22</height>
        </rect>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>291</width>
        <height>351</height>
       </rect>
      </property>
      <property name="title">
       <string>套装效果</string>
      </property>
      <widget class="QComboBox" name="cb_aeffect1">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>50</y>
         <width>151</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cb_aeffect2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>110</y>
         <width>151</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>1：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>2：</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>说明</string>
     </attribute>
     <widget class="QToolBox" name="toolBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1281</width>
        <height>351</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>98</width>
         <height>28</height>
        </rect>
       </property>
       <attribute name="label">
        <string>15s模型</string>
       </attribute>
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1271</width>
          <height>281</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="plainText">
         <string>15秒模型
======================
由 小遥想吃红豆沙 首发于米游社（https://bbs.mihoyo.com/ys/article/3218704）

摘录描述如下
------------
Q：为什么选择15秒爆发模型？

A：因为大多数角色BUFF持续12~15秒。 大多数角色大招CD都在15秒。可以看出，原神设计上就是在15秒上下浮动，大致分为12秒 15秒 18秒 21秒档位，而组队起来的时候，往往以15秒的角色为核心模型，所以选择15秒模型最公平也最符合实战情况。

——

Q：为什么选择15秒爆发模型？

A：原神这个游戏，因为刷怪地点不一样，还有场景技能，导致很少会有连续几分钟输出的情况，常常在一轮输出之后处于躲避技能，拉怪，走位。这个时候很显然，爆发模型 比 完全的持续输出更符合实际情况。</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>98</width>
         <height>28</height>
        </rect>
       </property>
       <attribute name="label">
        <string>使用说明（点我）</string>
       </attribute>
       <widget class="QPlainTextEdit" name="plainTextEdit_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1271</width>
          <height>281</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="plainText">
         <string>使用说明
======================
伤害计算
-----------
目前伤害只考虑伤害公式的前三个乘区以及技能倍率和技能次数的影响
元素反应以及敌人的各项属性均不计入（温蒂的扩散反应纳入计算）
阿贝多--目前是否速切都不计普攻


重要影响参数（需定义数据文件, 不建议随便尝试）
-----------
1. 人物的普攻，元素战技，元素爆发和重击的攻击轮数
2. 普攻/元素战技/元素爆发/重击的 具体组成
3.人物/武器特效的覆盖系数（因程序没有时间轴，特效效果用覆盖系数进行简化，相当于倍率）

速切
-----------
勾中时，普攻伤害不计入结果

日志
----------
可追踪具体每个技能分项的计算参数和人物信息，便于检查跟踪，计算时自动重置

======================
圣遗物主词条
------------
通过穷举完成，可通过勾选增加/减少选项
计算时候会自动存档
======================
圣遗物副词条
------------
也可输入其他人物带来的增益，比如班尼特可以调整固定攻击增益，圣遗物效果可调整属伤加成
计算时候会自动存档
</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1337</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
